code/game/machinery/washing_machine.dm

```diff
@@ -268,6 +268,13 @@ GLOBAL_LIST_INIT(dye_registry, list(
 		slowdown = SHOES_SLOWDOWN
 		new /obj/item/restraints/handcuffs(loc)
 	..()
+	
+/obj/item/clothing/shoes/clown_shoes/machine_wash(obj/machinery/washing_machine/WM)
+	if(istype(src, /obj/item/clothing/shoes/clown_shoes/prison))
+		new /obj/item/restraints/handcuffs(loc)
+	new /obj/item/clothing/shoes/clown_shoes(loc)
+	qdel(src)
+	..()
```

Comment:
```suggestion
	. = ..()
	qdel(src)
```
* Preserve the return value.
* Don't call for further behavior after setting the source for deletion.